bin/cron/process_foorm_data

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env ruby
+
+require_relative '../../lib/cdo/redshift'
+require_relative 'only_one'
+abort 'Script already running' unless only_one_running?(__FILE__)
+
+require_relative '../../dashboard/config/environment'
+
+S3_BUCKET_NAME = 'cdo-data-sharing-internal'
+aws_account_id = Aws::STS::Client.new.get_caller_identity.account
+REDSHIFT_S3_ACCESS_ROLE_ARN = "arn:aws:iam::#{aws_account_id}:role/redshift-s3"
+
+SUBMISSIONS_FROM_FILE_NAME = 'submissions.csv'
```

Comment:
These constant names were initially a bit confusing (because we have to write *to* the csv files first before reading *from* them).  Instead of "from", "to", maybe using "csv" and "redshift" would be clearer?  Up to you.

Reply:
mmm good point. I thought to/from would make things more clear, but yeah that's not the case. I'll try something else.